gradle-witchcraft-logging/src/main/groovy/com/palantir/witchcraft/java/logging/gradle/testreport/FormattingTestReporter.java

```diff
@@ -61,6 +61,21 @@ private static final class FormattingTestResultsProvider implements TestResultsP
                             writer.write("\n");
                         }
                         : Writable.NOP));
+        private static final BiConsumer<String, Writer> LINE_PROCESSOR = (line, outputWriter) -> {
+            try {
+                PARSER.tryParse(line)
+                        .orElseGet(() -> out -> {
+                            try {
+                                out.write(line);
+                            } catch (IOException e) {
+                                throw new RuntimeException(e);
```

Comment:
nit: I'm surprised error-prone doesn't error on this

```suggestion
                                throw new UncheckedIOException(e);
```